emiliopavia/io-app
# Add the new profile screen

⚠️ This pr depends on https://github.com/emiliopavia/io-app/pull/3 ⚠️
Please review https://github.com/emiliopavia/io-app/pull/3 first in order to avoid duplicate review work.

## Short description
This PR implements the new profile screen, replacing the current one for the `ROUTES.PROFILE_DATA` route when the feature flag is enabled.

## List of changes proposed in this pull request
- Several selectors have been implemented to extract user info from the profile's `pot`
- Implemented `UserProfileScreen` and added to the `ProfileNavigator` 

## How to test
Enable the `NEW_PROFILE_SCREEN` feature flag, go to the Profile tab and tap on "I tuoi dati" to see the new screen design.

| Before | After |
| -------| ----- |
| ![Simulator Screen Shot - iPhone 12 - 2022-03-01 at 11 26 41](https://user-images.githubusercontent.com/467098/156152219-ab01f93b-9b9b-4f00-86f5-f7d7a5906aa0.png) | ![Simulator Screen Shot - iPhone 12 - 2022-03-01 at 11 25 37](https://user-images.githubusercontent.com/467098/156152242-1f6b83f4-70d9-4f60-833b-ee4904e7f7c7.png) |
